torch_tvm/custom_tvm_ops/topi/contrib/quantize.cc

```diff
@@ -0,0 +1,80 @@
+#include <dmlc/logging.h>
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/util.h>
+#include <cmath>
+#include <iostream>
+
+namespace tvm {
+namespace contrib {
+
+using namespace runtime;
+
+TVM_REGISTER_GLOBAL("tvm.contrib.find_minmax")
+    .set_body([](TVMArgs args, TVMRetValue* ret) {
+      DLTensor* input = args[0];
+      DLTensor* data_min = args[1];
+      DLTensor* data_max = args[2];
+      // calculate the data_min and data_max
+      CHECK(input->strides == nullptr) << "find_minmax does not support the dltensor with strides";
+      auto data_ptr = static_cast<float *>(input->data);
+      int m = input->shape[0];
+      int n = input->shape[1];
+      float d_min = data_ptr[0];
+      float d_max = data_ptr[0];
+      for (int i = 0; i < m; i++) {
+        for (int j = 0; j < n; j++) {
+            d_min = std::min(data_ptr[i*n + j], d_min);
+            d_max = std::max(data_ptr[i*n + j], d_max);
+        }
+      }
+      auto out_ptr_min = static_cast<float *>(data_min->data);
```

Comment:
reinterpret_cast?

Reply:
I think it is void pointer. Static_cast should be better?

Reply:
IIRC, it should be void pointer.

Reply:
I suppose static_cast is safer

Reply:
Yes